### PR TITLE
Rename nix_installer_and_cache to cache_nix

### LINKED ISSUE #6550: CI - rename `nix_installer_and_cache` to `cache_nix` in elixir workflow
We recently renamed composite action that caches our Nix dependencies in this pull request https://github.com/build-trust/ockam/pull/6532, we should also rename the [elixir workflow that uses cache action](https://github.com/build-trust/ockam/blob/develop/.github/workflows/elixir.yml#L62) from 
```yaml
- uses: ./.github/actions/nix_installer_and_cache
```
to
```yaml
- uses: ./.github/actions/cache_nix
```
---

<i>

We love helping new contributors! ❤️
If you have questions or need help as you explore, please join us on [Discord](https://discord.gg/bewvnm6zqS). If you're looking for other issues to contribute to, please checkout our [good first issues](https://github.com/build-trust/ockam/labels/good%20first%20issue).

</i>

### PR DESCRIPTION
**Pull request by DIlkhush00**
## Current behavior

Current composite action in elixir workflow:
```YAML
 - uses: ./.github/actions/nix_installer_and_cache
 ```

## Proposed changes

Rename `nix_installer_and_cache` to `cache_nix` in elixir workflow. Composite action after the change: 
```YAML
- uses: ./.github/actions/cache_nix
```
Closes the issue #6550 

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] There are no Merge commits in this Pull Request. Ockam repo maintains a linear commit history. We merge Pull Requests by rebasing them onto the develop branch. Rebasing to the latest develop branch and force pushing to your Pull Request branch is okay.
- [x] I have read and accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have read and accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/.github/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam/blob/develop/.github/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam](https://github.com/build-trust/ockam) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam/edit/develop/.github/CONTRIBUTORS.csv) file in the github web UI and create a separate Pull Request, this will mark the commit as verified.

### CODE REVIEW

**Review by metaclips — COMMENTED** [review]
@DIlkhush00 Thank you for spending time on this, before we can merge please

Accept the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding your Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam/blob/develop/.github/CONTRIBUTORS.csv) file in a separate new pull request to the [build-trust/ockam](https://github.com/build-trust/ockam) repository.
The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam/edit/develop/.github/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

**Comment by DIlkhush00** [comment]
done 👍🏻 #6612

**Comment by metaclips** [comment]
Thanks @DIlkhush00. We should review soon

**Review by metaclips — APPROVED** [review]

**Pull request merged**
